apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -650,7 +650,11 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
     correct_answers = data["correct_answers"][answer_name]
     blocks = data["params"][answer_name]
 
-    if grading_method in [GradingMethodType.RANKING, GradingMethodType.DAG]:
+    if grading_method in [
```

Comment:
Looks like this list is repeated in this PR. Would be better to declare it a `frozenset` at the top of the file and reuse.

Reply:
Good catch. Fixed.